docs/src/expl_damage.md

```diff
@@ -0,0 +1,59 @@
+# [Damage in peridynamics formulations](@id expl_dmg)
+
+## Bond-based formulation
+
+```math
+ \boldsymbol{f} = d^{ij} \, c \, \varepsilon^{ij} \, \boldsymbol{n} \; .
+```
+
+with
+
+| size | symbol |      unit |
+|:--------|:-------------|:------------|
+| bond failure |      $d^{ij} \in \{0;1\}$      | $[-]$ |
+
+In this expression, bond failure $d^{ij}$ represents whether the bond between points $i$ and $j$ is intact ($d^{ij}=1$)
+or damaged ($d^{ij}=0$).
+
+## Ordinary state-based formulation
+
```

Comment:
Maybe add a TODO to all these subsections, that we do not forget to include the equations for the different formulations with the damage quantity $d^{ij}$
```suggestion
TODO
```